packages/@aws-cdk/aws-iam/lib/user.ts

```diff
@@ -159,16 +162,18 @@ export class User extends Resource implements IIdentity, IUser {
         return { statementAdded: true, policyDependable: this.defaultPolicy };
       }
 
-      public addToGroup(_group: IGroup): void {
-        throw new Error('Cannot add imported User to Group');
+      public addToGroup(group: IGroup): void {
+        this.groups.push(group.groupName);
```

Comment:
I don't think this is sufficient. If the `User` is being created you can add it to a group in this way, but fairly sure that's not how you do it if the User already exists.